v19.2/cockroachcloud-troubleshooting-page.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Troubleshoot CockroachCloud
+summary: The SQL Users page displays a list of SQL users who can access the cluster.
+toc: true
+build_for: [cockroachcloud]
+---
+
+This page describes common CockroachCloud errors and their solutions.
+
+## Connection errors
+
+### Wrong connection string
+
+The following error is displayed on the terminal if you use the incorrect connection string while trying to connect to a cluster:
+
+~~~ shell
+Error: x509: certificate signed by unknown authority
```

Comment:
I wouldn't say that I'm particularly well versed in the errors generated when connecting to a cluster but this error sounds like it would come from not having the correct `certs` directory specified?

The error below, ` i/o timeout`, could come from copying the connection string incorrectly as well.

Reply:
Nope..the error you get if the path is incorrect is:

~~~
Error: open amruta-192-gs-ca.crt: no such file or directory
Failed running "sql"
~~~

But this could be confusing as well. Adding this scenario to the doc.

The errors that rise out of copying connection string incorrectly are tricky. It depends on what part of the connection string they got wrong. I also don't anticipate the users copying the connection string incorrectly since they just need to press the copy button and we will copy it correctly for them.

Reply:
Done :)